app/assets/stylesheets/modules/sul-footer.scss

```diff
@@ -0,0 +1,59 @@
+#su-content {
+  padding-bottom: $footer-height;
+}
+
+#sul-footer-container {
+  background-color: $sul-footer-bg-color;
+  -webkit-box-shadow: 0 4px 8px -8px rgba(0, 0, 0, 0.2) inset;
```

Comment:
Color literals like `rgba(0, 0, 0, 0.2)` should only be used in variable declarations; they should be referred to via variable everywhere else.